MANA_VERSION

```diff
@@ -0,0 +1 @@
+mana/0.0.1
```

Comment:
can this PR be separated from json rpc?

Reply:
Sure. It will be after rebase. Hence the tags in progress.